src/org/zaproxy/zap/extension/websocket/WebSocketAPI.java

```diff
@@ -431,6 +438,24 @@ public ApiResponse handleApiView(String name, JSONObject params) throws ApiExcep
                 LOG.error(e.getMessage(), e);
                 throw new ApiException(ApiException.Type.INTERNAL_ERROR);
             }
+        } else if (VIEW_BREAK_TEXT_MESSAGE.equals(name)) {
+            ExtensionBreak extBreak = Control.getSingleton().getExtensionLoader().getExtension(ExtensionBreak.class);
+            if (extBreak != null) {

```

Comment:
This could check for null instead, it would reduce nesting. (Same comment in the action.)